src/App.css

```
@@ -164,65 +168,88 @@ form {
   border-color: #2f4f4f;
   border-style: solid;
   border-width: 1px;
-  width: 40%;
+  width: 50%;
   box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
   padding: 40px;
   transition: transform 0.3s ease-in-out;
   display: flex;
   flex-direction: column;
   justify-content: center;
-
-  @media only screen and (max-width: 800px) {
-    width: 80%  }
+}
+@media (max-width: 768px){
+  .container {
+    width: 50%;
+  }
+  .sheSharp {
+    font-size: 18px;
+    font-weight: 500;
+  }
+}
+@media(max-width: 475px){
+  .container{
+    width: 70%;
+  }
+  .sheSharp{
+    font-size: 16px;
+    font-weight: 500;
+  }
 }
 
 .container:hover {
   transform: scale(1.1);
 }
 
-.input-field {
-  margin-bottom: 10px;
+.form-align {
+  display: flex;
+  flex-direction: column;
+  max-width: 300px; /* Adjust as needed */
+  margin: auto;
+  font-size: 24px;
+}
+.form-align h3{
+  margin-bottom:10px;
+}
+label {
+  margin-top:5px;
+ display: flex;
+}
+.input-field,
+.message-input {
+  width: 500px;
   padding: 8px;
+  margin-bottom: 15px;
   border: 1px solid #ddd;
   border-radius: 5px;
 }
 
-.message-input {
-  height: 80px;
-}
-
 /* FOOTER STYLES */
 
 .footer {
   background-color: #66ccb0;
   max-width: 100%;
-  margin-left: -2px;
-  padding: 5px 10px;
   display: flex;
   justify-content: center;
-
-  width: 100%;
   color: #4fb39b;
   font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 20px;
 }
 
 .footer-logo {
-  width: 70px;
+  width: 100px;
+  margin-top: 10px;
+  margin-bottom: 10px;
 }
 
 .logo_links {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  width: 85%;
-  margin: 20px auto;
-}
+display: flex;
+align-items: center;
+justify-content: space-between;
 
-ul {
-  list-style-type: none;
-  padding-left: 15px;
 }
-
+.logo_links ul {
```

```suggestion
.logo-links ul {
```
For consistency can we update this to a dash instead of an underscore?